internal/controllers/clusterextension_controller.go

```diff
@@ -466,32 +473,46 @@ type DefaultInstalledBundleGetter struct {
 	helmclient.ActionClientGetter
 }
 
-func (d *DefaultInstalledBundleGetter) GetInstalledBundle(ctx context.Context, ext *ocv1alpha1.ClusterExtension) (*ocv1alpha1.BundleMetadata, error) {
+type InstalledBundle struct {
+	ocv1alpha1.BundleMetadata
+	Image string
+}
```

Comment:
Any particular reason why this type is added and used only within the `ClusterExtension` controller instead of extending the `ocv1alpha1.BundleMetadata` type to also include the image?

Reply:
@joelanford didn't want to update the external API (which is what would happen if `ocv1alpha1.BundleMetadata` is updated, as it's part of the `ClusterExtension.status`. We didn't want to put the Image location in there, only the metadata.

Reply:
Makes sense. I had a brief chat with @joelanford yesterday and we touched on this and the desire for the public api to have an explicit design for how we include references without constraining ourselves to specific assumptions.